deploy/kubernetes/overlays/stable/kustomization.yaml

```diff
@@ -4,7 +4,7 @@ bases:
   - ../../base
 images:
   - name: amazon/aws-efs-csi-driver
-    newTag: v1.3.2
+    newTag: v1.3.3
```

Comment:
for the new release process we want to update kustomize only after the image has been pushed https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/docs/RELEASE.md (so we will need 2 PRs for every release)

Reply:
I guess for this repo since the helm release action acts on the relkease branch instead of master branch we also have t odelay helm changes until after image build